### DISCUSSION TITLE
Lane Connectors

### DISCUSSION THREAD

**Discussion by waso99** [CitiesSkylinesMods/TMPE#1723] [Ideas]
Hi,
Copy and paste Lane Connectors, is good idea.

**Comment by krzychu124** [comment]
You can already copy and paste whole intersections with TM:PE settings if you use Move it mod

**Reply by waso99** [reply]
But this is not solution!!!

**Reply by krzychu124** [reply]
The problem is, it's not trivial to implement but we are aware it's one of things a lot of users request, so I'm not saying it will never be added, but no ETA.
